main/src/cgeo/geocaching/maps/CGeoMap.java

```diff
@@ -423,6 +423,12 @@ public void onCreate(final Bundle savedInstanceState) {
 
         ArrayList<Location> trailHistory = null;
 
+        final Geopoint savedCoords = mapOptions.coords;  // remember for centerMap
```

Comment:
Is it possible to move the whole block down to fix the codacy issue?

Reply:
done